etc/config/hook.defaults.properties

```diff
@@ -0,0 +1,12 @@
+# Default settings for Hook
+
+supportsBinary=false
+interpreted=true
+supportsExecute=false
+versionFlag=--version
+compilerType=hook
+needsMulti=false
+compilers=hook
+defaultCompiler=hook
```

Comment:
Also, a better id for a default compiler is something not as generic, so `hookdefault` would be better :)

Reply:
No problem.